packages/aila/src/core/chat/AilaStreamHandler.ts

```diff
@@ -172,14 +172,6 @@ export class AilaStreamHandler {
       ...this._chat.aila.document.content,
     };
 
-    if (
```

Comment:
Since we now have agents for these, if the categoriser fails to categorise, we can attempt in the agent model -- or the agent can ask the user for more information